src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -186,11 +194,15 @@
                                                :from 0
                                                :size 10}))))}
         :typeaheadDisplay (fn [result]
-                            ;; we intentionally do not encode the result value here, because it is already
-                            ;; encoded from the server.
-                            (.text (js/$ (str "<div>" (aget result "value") "</div>"))))
+                            (let [sugg-object (js->clj (aget result "value"))
+                                  suggestion (get sugg-object "suggestion")]
+                              (.text (js/$ (str "<div>" suggestion "</div>")))))
```

Comment:
we *can* do `(js/$ (str "<div>" (encode suggestion) "</div>"))`, and it seems to work for the end user ... but it feels like we should use *either* `jQuery.text` or `encode`, and not both.